authentication/authentication-internal-service/src/main/java/io/mosip/authentication/internal/service/integration/RestHelperImpl.java

```diff
@@ -198,9 +188,9 @@ private void checkErrorResponse(Object response, Class<?> responseType) throws R
 						mapper.readValue(responseNode.toString().getBytes(), responseType));
 			}
 		} catch (IOException e) {
-			mosipLogger.error(IdAuthCommonConstants.SESSION_ID, CLASS_REST_HELPER, REQUEST_SYNC_RUNTIME_EXCEPTION,
-					THROWING_REST_SERVICE_EXCEPTION + "- UNKNOWN_ERROR - " + e);
-			throw new RestServiceException(IdAuthenticationErrorConstants.UNABLE_TO_PROCESS, e);
+			mosipLogger.warn(IdAuthCommonConstants.SESSION_ID, CLASS_REST_HELPER, REQUEST_SYNC_RUNTIME_EXCEPTION,
+					THROWING_REST_SERVICE_EXCEPTION + "- UNKNOWN_ERROR - "
+							+ StringUtils.substring(e.getMessage(), 0, 500));

```

Comment:
e.getMessage() is just the error message, it need not be limited
Pls change the log level to error